uvicorn/main.py

```diff
@@ -263,7 +263,10 @@ def run(self):
         if not config.loaded:
             config.load()
 
-        self.loop = config.loop_instance
+        if not config.loop_setup:
+            config.setup_event_loop()
```

Comment:
We won't need to branch here, we'll just call `config.setup_event_loop()` whatever.